be/src/exprs/aggregate_functions.cpp

```diff
@@ -1241,6 +1241,38 @@ int64_t AggregateFunctions::hll_algorithm(const doris_udf::StringVal& src) {
     return (int64_t)(estimate + 0.5);
 }
 
+void AggregateFunctions::hll_raw_agg_init(
+        FunctionContext* ctx,
+        StringVal* dst) {
+    hll_union_agg_init(ctx, dst);
+}
```

Comment:
All calculations can be based on full type, so dst will be initted to a hll full type .